lib/Sema/TypeCheckDeclOverride.cpp

```diff
@@ -13,6 +13,7 @@
 // This file implements semantic analysis for declaration overrides.
 //
 //===----------------------------------------------------------------------===//
+#include <iostream>
```

Comment:
Do we need this? I'm guessing you needed this import for `std::cerr` so you can use print statements for debugging, however you can use `llvm::errs()` instead.

Reply:
Oh yes I forgot to remove that. And thanks for the `llvm::errs()` suggestion. I was not aware of it. :)